lib/fennec/stun/auth.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Fennec.STUN.Auth do
```

Comment:
But Fennec is all about STUN and TURN. Again, it reminds me of this other server, and it was really confusing for me at a time.